### DISCUSSION TITLE
An escape sequence instead of the tag's bracket

### DISCUSSION THREAD

**Discussion by salmonsenya** [mganss/HtmlSanitizer#505] [Q&A]
Hello! I have encountered the  following problem: I expect text
`It is part of the <а href="https://example.com/upload/images/31271_offer.pdf"> user agreement</а>`

to look exactly the same after sanitization (a, href are allowed). Instead it contains escape sequences:

`It is part of the &lt;а href="https://example.com/upload/images/31271_offer.pdf"&gt; user agreement`

How may I avoid this behavior?

**Comment by mganss** [comment]
The `a` in your example is not a latin a (0x61) but a cyrillic a (0x430).

**Reply by salmonsenya** [reply]
Omg, so stupid of me! Thank you so much
